database/src/main/java/com/firebase/ui/database/FirebaseArray.java

```diff
@@ -24,7 +24,7 @@
 /**
  * This class implements an array-like collection on top of a Firebase location.
  */
-class FirebaseArray implements ChildEventListener {
+public class FirebaseArray implements ChildEventListener {
```

Comment:
I so far consider `FirebaseArray` to *not* be a part of the FirebaseUI public API. It's fine to discuss whether my consideration is valid, but that discussion is best had in an issue and not in a PR. See for example #206 and #337 (which I haven't responded to yet).